x-pack/plugins/maps/common/descriptor_types/sources.ts

```diff
@@ -169,6 +169,7 @@ export type LayerDescriptor = {
   alpha?: number;
   id: string;
   label?: string | null;
+  labelsOnTop?: boolean;
```

Comment:
consider changing to `isLabelsOnTop`. it seems more common in use in Map for booleans.

`is[Visible|Ordinal|Categorical|Fieldaware|ExtentAware|GeoGridPrecisionAware|LayerLoading|...]`